app/messages.py

```diff
@@ -75,7 +75,10 @@
 # Update
 NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT = {"message": "No data for updating"
                                          "profile was sent"}
-
+USER_ENTERED_CURRENT_PASSWORD_FOR_NEW_PASSWORD = {"message": "New password cannot be the same "
```

Comment:
Can you remove this since it is not invoked anymore?

Reply:
Done!